lighthouse-core/lib/network-recorder.js

```diff
@@ -9,9 +9,16 @@ const log = require('lighthouse-logger');
 const NetworkRequest = require('./network-request.js');
 const EventEmitter = require('events').EventEmitter;
 
-/** @typedef {'requeststarted'|'requestfinished'} NetworkRecorderEvent */
+/**
+ * @typedef {{
+ *   requeststarted: [NetworkRequest],
+ *   requestfinished: [NetworkRequest],
+ * }} NetworkRecorderEventMap
```

Comment:
It could be, but the main benefit of the `@property` form is you can document the properties. This won't ever be used as the type for a real object, the properties are for type lookup by event name, so there wouldn't ever be anywhere those jsdoc strings would show up